app/src/main/java/com/github/se/bootcamp/ui/screens/ProfileScreen.kt

```diff
@@ -0,0 +1,309 @@
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+//noinspection UsingMaterialAndMaterial3Libraries
+import androidx.compose.material.Icon
+//noinspection UsingMaterialAndMaterial3Libraries
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.rememberImagePainter
+import com.github.se.bootcamp.R
+
+
+
+@Composable
+fun ProfileScreen(
+    userId: String,
+    userName: String,
+    profilePictureUrl: String?,
+    numberOfDays: Int,
+    lettersLearned: List<Char>,
+    easyExercises: Int,
+    hardExercises: Int,
+    dailyQuests: Int,
+    weeklyQuests: Int,
+    onSettingsClick: () -> Unit,
+    onHelpClick: () -> Unit,
+    onFriendsClick: () -> Unit,
+    onGraphClick: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(16.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        // Top row with Settings and Help buttons
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween
+        ) {
+            Button(onClick = onSettingsClick) {
+                Text("Settings")
+            }
+            Button(onClick = onHelpClick) {
+                Text("Help")
+            }
+        }
+
+        Spacer(modifier = Modifier.height(32.dp))
+
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            // User Info: Display Test ID and Test Name in a Column
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                //modifier = Modifier.fillMaxWidth()
```

Comment:
deally, we should avoid merging commented code into the main branch. Commented code tends to stay longer than expected and can accumulate technical debt. If this code is necessary for future work, it might be better to track it in a task or discussion, or remove it until it's ready for use.